R/pstr.R

```diff
@@ -28,19 +32,19 @@
 #'
 #' # Same
 #' pstr(companies, scenarios)
-pstr <- function(companies, scenarios) {
+pstr <- function(companies, scenarios, low_threshold = 30, high_threshold = 70) {
```

Comment:
In xctr we use 1/3 and 2/3. Do you plan to do that in a separate PR?

Reply:
@maurolepore I can see the `reductions` variable is not between 0 and 1. Hence the thresholds' values cannot be changed to 1/3 and 2/3. Please @lindadelacombaz correct me if I am wrong

Reply:
Yes true. I have asked @Tilmon if the reduction column are in percentage, and they are, so I have used 30% and 70% values.

Reply:
@maurolepore I will resolve this problem through this [issue](https://github.com/2DegreesInvesting/tiltIndicator/issues/165).